docs/deployment/add-ons/storage/fsx-for-lustre/guide.md

```diff
@@ -1,37 +1,77 @@
-# Using Amazon FSx as Persistent Storage with Kubeflow
++++
+title = "FSx for Lustre"
+description = "Use Amazon FSx as persistent storage with Kubeflow on AWS"
+weight = 20
++++
 
 This guide describes how to use Amazon FSx as Persistent storage on top of an existing Kubeflow deployment.  
```

Comment:
There is an intro in the quip doc for both efs and fsx but we can work on getting that in post release.

Reply:
@judyheflin Could you track this in your backlog

Reply:
Yep!